.circleci/config.yml

```diff
@@ -7,28 +7,22 @@ jobs:
     steps:
```

Comment:
Looking at this I like the bold changes. I think I get too timid with these kinds of chores, trying to keep them tiny in scope.